main.ts

```diff
@@ -1,11 +1,11 @@
 import fetch from 'node-fetch';
-import * as fs from 'fs';
+import {URLSearchParams} from 'url';

```

Comment:
This is the correct way to do URL parameters. FormData is for frontend. (this was also the example in the node-fetch docs)